artifactory/cli.go

```diff
@@ -2328,6 +2328,16 @@ func transferConfigCmd(c *cli.Context) error {
 		return nil
 	}
 
+	// Check if there is a configured user using default credentials in the source platform.
+	if isDefaultCredentials(sourceServerDetails.ArtifactoryUrl) {
+		defaultCredMsg := "The default 'admin:password' credentials are used by a configured user in your source platform.\n" +
+			"Those credentials will be transferred to your SaaS target platform.\n" +
+			"Are you sure you want to continue?"
+		if !coreutils.AskYesNo(defaultCredMsg, false) {
+			return nil
+		}
```

Comment:
```suggestion
		log.Output()
		log.Warn("The default 'admin:password' credentials are used by a configured user in your source platform.\n" +
			"Those credentials will be transferred to your SaaS target platform.")
		if !coreutils.AskYesNo("Are you sure you want to continue?", false) {
			return nil
		}
```

After the change:
<img width="910" alt="image" src="https://user-images.githubusercontent.com/11367982/179486131-e5e7db45-50da-4aaf-9b7f-b6f3de3abb56.png">